lib/Doctrine/Common/Collections/Expr/ClosureExpressionVisitor.php

```diff
@@ -103,6 +103,7 @@ public static function sortByField($name, $orientation = 1, ?Closure $next = nul
 
         return static function ($a, $b) use ($name, $next, $orientation) : int {
             $aValue = ClosureExpressionVisitor::getObjectFieldValue($a, $name);
+
```

Comment:
This seems incorrect

Reply:
If I revert the change - codesniffer screams:

```
FILE: lib/Doctrine/Common/Collections/Expr/ClosureExpressionVisitor.php
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 104 | ERROR | [x] Expected 1 lines after "return", found 0.
     |       |     (SlevomatCodingStandard.ControlStructures.ControlStructureSpacing.IncorrectLinesCountAfterControlStructure)
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```

Need to have a closer look  into the slevomat rules to figure out whats going wrong here

Reply:
Could you reproduce this and report it to `doctrine/coding-standard` maybe? We can roll with this patch meanwhile 👍

Reply:
Boiled it down to a simple example and open report here: https://github.com/slevomat/coding-standard/issues/635